### PR TITLE
docs: add recommended config of language to zh-cn

### PR DESCRIPTION
**Pull request by xyyuhans**
## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
    - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
    - [ ] `preview` URL is correct.
    - [ ] `preview` URL web site is rendered correctly.
    - [ ] Add a screenshot to `source/themes/screenshots`.
    - [ ] Screenshot filename is same as value of `name`.
    - [ ] Screenshot size is `800 * 500`.
    - [ ] Screenshot file format is `png`.
- [ ] I want to publish my plugin on Hexo official website.
    - [ ] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
- [x] Others (Update, fix, translation, etc...)

### CODE REVIEW

**Comment by curbengh** [comment]
> There are some themes using `zh-Hans` and `zh-Hant` for their i18n.

OP does mention "一般的", so it doesn't imply that "zh-CN" would always work. I'm assuming "zh-CN" is more common than "zh-Hans"?

I personally do recommend theme that uses "zh-Hans" to also have a "zh-CN" symlink for best compatibility.

**Comment by SukkaW** [comment]
@curbengh Currently the available i18n is provided and controlled by the theme, not by Hexo. So users should follow the documents provided by the theme.

**Comment by xyyuhans** [comment]
@SukkaW Update according to your reply.

File: source/zh-cn/docs/configuration.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ title: 配置
 `description` | 网站描述
 `keywords` | 网站的关键词。使用半角逗号 `,` 分隔多个关键词。
 `author` | 您的名字
-`language` | 网站使用的语言
+`language` | 网站使用的语言。对于简体中文用户来说，使用不同的主题可能需要设置成不同的值，请参考你的主题的文档自行设置，常见的有 `zh-Hans`， `zh-CN`。
```
Comment (SukkaW) [inline source/zh-cn/docs/configuration.md]:
```suggestion
`language` | 网站使用的语言，请遵循主题的使用文档或配置指南进行设置。
```

**Pull request merged**